src/Compilers/VisualBasic/Portable/Lowering/StateMachineRewriter/StateMachineFieldSymbol.vb

```diff
@@ -57,7 +58,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                    slotIndex:=slotindex,
                    accessibility:=accessibility,
                    isReadOnly:=isReadOnly,
-                   isShared:=isShared)
+                   isShared:=isShared,
+                   isSpecialNameAndRuntimeSpecial:=isSpecialNameAndRuntimeSpecial)
```

Comment:
1. why does this need to be done?
2. what impact does this have on any users programs?
3. this needs tests to validate behavior.